packages/fx-core/src/plugins/resource/bot/v2/plugin.ts

```diff
@@ -135,7 +135,12 @@ export class TeamsBotV2Impl {
     tokenProvider: TokenProvider
   ): Promise<Result<Void, FxError>> {
     const projectPath = checkPrecondition(Messages.WorkingDirIsMissing, inputs.projectPath);
-    const workingDir = path.join(projectPath, CommonStrings.BOT_WORKING_DIR_NAME);
+    const language = getLanguage(ctx.projectSetting.programmingLanguage);
+    const workingDir =
```

Comment:
There is similar logic of getting working dir in scaffold(), we can extract it as a separate method for single source of truth later.